### DISCUSSION TITLE
uniform pixel cropping

### DISCUSSION THREAD

**Discussion by Nezarod** [ImageMagick/ImageMagick#6114] [Help]
### ImageMagick version

7

### Operating system, version and so on

windows

### Description

sorry for my english - I translate through an online translator

is there any simple way to crop empty pixels - but so that cropping from all sides is uniform.
those. after cropping, the picture should remain in the center.

let me explain

if you use the function
magick.exe 1.png -trim +repage OUTPUT.PNG

then the picture above can be cropped, for example, by 100 pixels, and below by 50 pixels.
as a result, the image will be off-center.

how to make it so that if the picture is cut off from the bottom by 50 pixels, from the top it would also be cut off only by 50 pixels, and not by 100?

Thank you!

**Comment by fmw42** [comment]
I have a centertrim bash unix shell script that does that at http://www.fmwconcepts.com/imagemagick/centertrim/index.php.  But sorry, it does not work on DOS Windows, but should work using Windows Unix.

**Comment by Nezarod** [comment]
Thank you. already wrote it. also added centering without taking into account the shadow)
